scripts/pull_github_pr.sh

```diff
@@ -72,24 +73,21 @@ nr_commits=$(git log --pretty=oneline HEAD..FETCH_HEAD | wc -l)
 closes="${NL}${NL}Closes ${PROJECT}#${PR_NUM}${NL}"
 
 if [[ $nr_commits == 1 ]]; then
-	commit=$(git log --pretty=oneline HEAD..FETCH_HEAD | awk '{print $1}')
-	message="$(git log -1 "$commit" --format="format:%s%n%n%b")"
-	if ! git cherry-pick $commit
+	if ! git merge --log "$PR_REF"
 	then
-		echo "Cherry-pick failed. You are now in a subshell. Either resolve with git cherry-pick --continue or git cherry-pick --abort, then exit the subshell"
+		echo "Merge failed. You are now in a subshell. Either resolve with git merge --continue or git merge --abort, then exit the subshell"
 		head_before=$(git rev-parse HEAD)
 		bash
 		head_after=$(git rev-parse HEAD)
 		if [[ "$head_before" = "$head_after" ]]; then
 			exit 1
 		fi
 	fi
-	git commit --amend -m "${message}${closes}"
 else
-	git merge --no-ff --log=1000 FETCH_HEAD -m "Merge '$PR_TITLE' from $USER_NAME" -m "${PR_DESCR}${closes}"
+	git merge --no-ff --log=1000 FETCH_HEAD -m "Merge '$PR_TITLE' from $USER_NAME" -m "${PR_DESCR}"
 fi
 git commit --amend # for a manual double-check
-
+git notes add -f -m "${closes}"
```

Comment:
Is this recognized by github or is just for us?

Reply:
Just for us, GitHub will not show it in the UI